ncm-metaconfig/src/main/metaconfig/mailrc/pan/schema.pan

```diff
@@ -9,6 +9,7 @@ type mailrc_element = {
     'smtp-auth-user' ? string
     'smtp-auth-password' ? string
     'nss-config-dir' ? string
+    'ssl-verify' ? string
```

Comment:
Make this a choice type? `man 5 mailrc` isn't turning up much for me but Google suggests this can only take a limited set of strings which we could capture here to prevent typos?

Reply:
From `man 1 mailx`:

> Valid values are
> `strict` (fail and close connection immediately),
> `ask` (ask whether to continue on standard input),
> `warn` (print a warning and continue),
> `ignore` (do not perform validation).
> The default is `ask`.

Reply:
I have converted it to choice, as requested. Thx for the options pointer @jrha